SalesforceCommerceCloud/pwa-kit
# Add Shopper Experience hooks

# Description



The PR adds the two new Shopper Experience endpoints as hooks to `commerce-sdk-react-preview`.
#### Endpoint docs used by `usePage`
- https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-experience?meta=getPage
- https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperexperience.shopperexperience-1.html#getpage
#### Endpoint docs used by `usePages`
- https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-experience?meta=getPages
- https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperexperience.shopperexperience-1.html#getpages 
 



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added usePage and usePages hooks to hooks to `commerce-sdk-react-preview`.
- Added test pages to the `test-commerce-sdk-react` project.

# How to Test-Drive This PR

1. Run the `test-commerce-sdk-react` project.
```
npm run start --prefix packages/test-commerce-sdk-react/
```
2. Load the test project in your browser http://localhost:3000/ and click the button `Switch to RefArch`.
3. Navigate to the [useShopperExperience](http://localhost:3000/experience) page.

4. Verify we get the expected responses.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
